nibabel/tests/test_arrayproxy.py

```diff
@@ -57,6 +60,10 @@ def copy(self):
 
 class CArrayProxy(ArrayProxy):
     # C array memory layout
+    _default_order = 'C'
+
+
+class DeprecatedCArrayProxy(ArrayProxy):
```

Comment:
Timed out depraction again (as above)?

Reply:
This is specifically for testing. The warning/expiration will show up in `test_deprecated_order_classvar`. When we remove that function, we can remove this class. I'll add a comment...